rdr_service/alembic/versions/e4eedba5965f_.py

```diff
@@ -47,7 +47,7 @@ def upgrade_rdr():
     op.execute(
         "Update genomic_member_report_state INNER Join genomic_set_member "
         "On genomic_set_member.id = genomic_member_report_state.genomic_set_member_id "
-        "Set module = 'GEM', "
+        "Set module = 'gem', "
```

Comment:
Is this the backfill migration?

Reply:
No that is the original, did a global find and replace to better match what is expected from the MessageBroker. For the backfills on ptsc-1-test, I have been running local SQL scripts. Can revert since this migration will not get run anyway on any environment if needed.

Reply:
Sounds good, no need to revert, was just wondering what the migration was for.